.gitignore

```diff
@@ -44,4 +44,8 @@ node_modules
 /bazel-*
 CMakeLists.txt.user
 CMakeFiles
-__generated__
\ No newline at end of file
+__generated__
+
+/metrics/integration/query-tests/**/actual.json
+/metrics/integration/render-tests/**/actual.png
+/metrics/integration/render-tests/**/diff.png
```

Comment:
Can we ignore all actual.png and diff.png, cause there are same files under the test directory.

Reply:
I guess we could...

Reply:
I don't see them in my changed files usually, what is the test program that generates them?

Reply:
Let's add the other ignores in a different PR...

Reply:
For example `/Users/wei.chen/work/maplibre/test/fixtures/annotations/add_multiple ` has actrual.png and diff.png, after cpp unit test(mbgl-test).

Reply:
There is already a [test/fixture/.gitignore](https://github.com/maplibre/maplibre-native/blob/main/test/fixtures/.gitignore) that has them 
```
diff.png
actual.png
*.db
```